src/app/tests/suites/certification/PICS.yaml

```diff
@@ -3604,6 +3604,9 @@ PICS:
     - label: "Indicates that node is configured for test event triggers."
```

Comment:
Add blocking comment to https://github.com/CHIP-Specifications/chip-test-plans/pull/3970 to merge first